ctapipe/image/extractor.py

```diff
@@ -1051,7 +1052,10 @@ def _apply_second_pass(
         # Apply correction to 1st pass charges
         charge_1stpass = charge_1stpass_uncorrected * correction[selected_gain_channel]
 
-        camera_geometry = self.subarray.tel[tel_id].camera.geometry
+        # do it in TelescopeFrame
```

Comment:
That comment is not very useful

Reply:
I guess that was just a note for myself😅